src/platform.ts

```diff
@@ -222,7 +222,7 @@ export function urlMatches(urlString: string, match: types.URLMatch | undefined)
     return true;
   if (helper.isString(match))
     match = helper.globToRegex(match);
-  if (match instanceof RegExp)
+  if (match instanceof RegExp || match.constructor.name === "RegExp")
```

Comment:
As mentioned here, https://stackoverflow.com/questions/4339288/typeof-for-regexp#comment27695426_4339326, underscore.js uses:
```javascript
Object.prototype.toString.call(obj) == '[object RegExp]'
```

Thoughts?